src/app/modules/shared-components/components/access-code-view/access-code-view.component.html

```diff
@@ -11,12 +11,12 @@
     <span class="icon">
       <i class="fa fa-font"></i>
     </span>
-    <alg-button
+    <p-button
+      styleClass="alg-button p-button-rounded"
       icon="fa fa-check"
       [label]="buttonLabel"
-      class="p-button-rounded"
       [disabled]="input.value === '' || state === 'loading'"
-      (click)="onClickAccess()"
-    ></alg-button>
+      (onClick)="onClickAccess()"
+    ></p-button>
```

Comment:
The don't look the same, is that intended?
Before:
![image](https://user-images.githubusercontent.com/16257159/168767906-4eed4b38-d7ce-4905-99b4-e130c8e463dc.png)

After:
![image](https://user-images.githubusercontent.com/16257159/168767995-4042f3aa-81a4-4f45-bf90-f50967e32d11.png)